index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>my-portfolio</title>
+</head>
+<body>
+    <div class="container">
+        <header class="navbar">
+            <a href="#">
+                <img class="my-logo" src="images/My Logo.png" alt="logo">
+            </a>
+            <a href="#">
+                <img class="menu" src="images/union.png" alt="union">
+            </a>
+        </header>    
+        <section class="main-section">  
+            <br>  
+            <h1 class="main-text">I’m Godanah
+                Glad to see you!</h1>
+            <p class="content">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <br>
+            <p>LET'S CONNECT</p>
+            <br>
+            <ul class="list-group">
+                <a href="#"><li class="list-group-item"><img src="images/vector-1.png" alt="vector1"></li></a>
+                <a href="#"><li class="list-group-item"><img src="images/Vector-2.png" alt="vector2"></li></a>
+                <a href="#"><li class="list-group-item"><img src="images/Vector-3.png" alt="vector3"></li></a>
+                <a href="#"><li class="list-group-item"><img src="images/Vector-4.png" alt="vector4"></li></a>
+                <a href="#"><li class="list-group-item"><img src="images/Vector-5.png" alt="vector5"></li></a>
+            </ul>
+        </section>
```

Comment:
- [ ] It would be better if you wrap these lines with a `main` tag in order to make your code Semantic 😃. Please see this [link ](https://www.freecodecamp.org/news/semantic-html5-elements/) in order to get more information about Semantic HTML.

Reply:
Well solved